cmd/uhc/token/cmd.go

```diff
@@ -142,58 +127,50 @@ func run(cmd *cobra.Command, argv []string) {
 	parser := new(jwt.Parser)
 	_, parts, err := parser.ParseUnverified(selectedToken, jwt.MapClaims{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't parse token: %v\n", err)
-		os.Exit(1)
+		return fmt.Errorf("Can't parse token: %v", err)
 	}
 	encoding := base64.RawURLEncoding
 	header, err := encoding.DecodeString(parts[0])
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't decode header: %v\n", err)
-		os.Exit(1)
+		return fmt.Errorf("Can't decode header: %v", err)
 	}
 	payload, err := encoding.DecodeString(parts[1])
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't decode payload: %v\n", err)
-		os.Exit(1)
+		return fmt.Errorf("Can't decode payload: %v", err)
 	}
 	signature, err := encoding.DecodeString(parts[2])
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't decode signature: %v\n", err)
-		os.Exit(1)
+		return fmt.Errorf("Can't decode signature: %v", err)
 	}
 
 	// Print the data:
 	if args.header {
 		err = dump.Pretty(os.Stdout, header)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Can't dump header: %v\n", err)
-			os.Exit(1)
+			return fmt.Errorf("Can't dump header: %v", err)
 		}
 	} else if args.payload {
 		err = dump.Pretty(os.Stdout, payload)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Can't dump payload: %v\n", err)
-			os.Exit(1)
+			return fmt.Errorf("Can't dump payload: %v", err)
 		}
 	} else if args.signature {
 		err = dump.Pretty(os.Stdout, signature)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Can't dump signature: %v\n", err)
-			os.Exit(1)
+			return fmt.Errorf("Can't dump signature: %v", err)
 		}
 	} else {
-		fmt.Fprintf(os.Stdout, "%s\n", selectedToken)
+		fmt.Fprintf(os.Stdout, "%s", selectedToken)
```

Comment:
I think here we want to preserve the `\n`.

Reply:
Fixed!